homestar-runtime/src/settings/libp2p_config.rs

```diff
@@ -128,6 +130,8 @@ impl Default for Libp2p {
         Self {
             announce_addresses: Vec::new(),
             dht: Dht::default(),
+            // https://github.com/libp2p/rust-libp2p/pull/4967
+            idle_connection_timeout: Duration::new(10, 0),
```

Comment:
Could we add this to `homestar-runtime/config/defaults.toml`?

Reply:
```toml
[node.network.libp2p]
idle_connection_timeout = 10
listen_address = "/ip4/0.0.0.0/tcp/0"
node_addresses = []
announce_addresses = []
transport_connection_timeout = 60
max_connected_peers = 32
max_announce_addresses = 10
```